protocol/daemons/liquidation/client/client.go

```diff
@@ -16,6 +17,8 @@ import (
 	satypes "github.com/dydxprotocol/v4-chain/protocol/x/subaccounts/types"
 )
 
+var nextKeyToFetch []byte
```

Comment:
Small suggestion: I think defining this as a variable [here](https://github.com/dydxprotocol/v4-chain/blob/a62e329fe1dc6befcc53caf9803abf4125160697/protocol/daemons/liquidation/client/client.go#L59) and then passing it in as a function argument to `RunLiquidationDaemonTaskLoop` would avoid needing a global variable and be cleaner